### DISCUSSION TITLE
I'm bignner_ meteoroligical forcing data sources

### DISCUSSION THREAD

**Discussion by Mirylis** [NASA-LIS/LISF#1511] [Q&A] [answered]
I am a new user of LISF, and currently, I am trying to understand LISF. I started by following the test case instructions, but unfortunately, I did so without a good understanding. So, I decided to read the documentation, and as I began to understand, I now have a question about the meteorological forcing data sources. I didn't really understand if LIS can provide the forcing data, which I can simply write into the LDT config entry, or if I should look for and download it myself as a files, as demonstrated in the test case. and if it so I want to know how to get it.

**Comment by MariRiba** [comment, marked as answer]
Hi 
I am not an expert but I think you can download and use your own meteorological forcings. In the test cases, they have used NLDAS but you can use GDAS , TRMM (for precipitation) etc which I am using. How to use forcing is given in the documentation [here](https://nasa-lis.github.io/LISF/LIS_users_guide/LIS_users_guide.html#ssec_driveropts) and [here](https://nasa-lis.github.io/LISF/LIS_users_guide/LIS_users_guide.html#ssec_forcings) . Have a look , Hope it helps .

**Comment by jvgeiger** [comment]
Hello.  Thanks @MariRiba.  You are correct.

@Mirylis, the LIS Users Guide lists all the meteorological forcing data sources that LIS can read.  You must download the data and then tell LIS which data source to use (and where you put the files) by setting entries in the LIS configuration file (lis.config).

Thanks.
